sdk/identity/identity/test/public/node/multiTenantAuthentication.spec.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { MsalTestCleanup, msalNodeTestSetup } from "../../node/msalNodeTestSetup";
+import { Recorder, env } from "@azure-tools/test-recorder";
+import { createHttpHeaders, createPipelineRequest } from "@azure/core-rest-pipeline";
+
+import { ClientSecretCredential } from "../../../src/credentials/clientSecretCredential";
+import { Context } from "mocha";
+import { IdentityClient } from "../../../src/client/identityClient";
+import { assert } from "@azure/test-utils";
+
+describe("MultiTenantAuthentication", function () {
+  let cleanup: MsalTestCleanup;
+  let recorder: Recorder;
+
+  beforeEach(async function (this: Context) {
+    const setup = await msalNodeTestSetup(this.currentTest);
+    cleanup = setup.cleanup;
+    recorder = setup.recorder;
```

Comment:
style nit: something I figured out recently is you can use destructuring in these cases still:

```suggestion
   ({cleanup, recorder} = await msalNodeTestSetup(this.currentTest));
```